kong/core/router.lua

```diff
@@ -659,16 +660,12 @@ function _M.new(apis)
       return nil
     end
 
-    local new_uri
-    local uri_root = uri == "/"
+    local uri_root = request_uri == "/"
```

Comment:
yes, i spose this comment was in reference to the desire to have some more commentary on this module in general. certainly not a merge blocker though :)